content/en/docs/gke/deploy/management-setup.md

```diff
@@ -6,6 +6,14 @@ weight = 3
 
 This guide describes how to setup a management cluster which you will use to deploy one or more instances of Kubeflow.
 
+For understanding questions like:
+
+* Where is kfctl?
+* Why do we need an extra management cluster?
+
+There are some early explanations in [this GitHub issue](https://github.com/kubeflow/gcp-blueprints/issues/123).
```

Comment:
Would it be possible to provide a short version of the answer to those questions and then link to the issue for further explanation?

Reply:
Good point! I added a short version.